lib/matplotlib/path.py

```diff
@@ -558,14 +558,12 @@ def intersects_bbox(self, bbox, filled=True):
         :class:`~matplotlib.transforms.Bbox`.
 
         *filled*, when True, treats the path as if it was filled.
-        That is, if one path completely encloses the other,
-        :meth:`intersects_path` will return True.
+        That is, if the path completely encloses the bounding box,
+        :meth:`intersects_bbox` will return True.
+
+        The bounding box is always considered filled.
```

Comment:
Is there a way to not change this behavior?  This is a public method on a public so we must be careful about changing API.  You will be surprised what behavior users have found and rely on :smiling_imp: .

On the other hand, the docstring was clearly a copy-paste from the method above and not quite right and I think it is reasonable to interpret 'intersects bbox'  as 'is in or crosses boundary of bbox'.  It that case this needs to be documented as an API change in https://github.com/matplotlib/matplotlib/tree/master/doc/api/api_changes